### DISCUSSION TITLE
Caused issue with X52 Hotas

### DISCUSSION THREAD

**Discussion by enigmaticzebra** [nefarius/HidHide#36] [Q&A] [answered]
After installing this caused windows 10 to no longer recognise my x52 as game contoller, but still showed in device manager, had to reinstall windows to resolve the issue as forewarning to others and/or the dev/s to look into

**Comment by nefarius** [comment, marked as answer]
That's major overkill though. Why no backup? Why not seek support before wiping the system? So far we had **a 100% success** with helping everyone who borked their HID drivers one way or the other so no real need for a forewarning 😛 Now that it's wiped the report is kinda pointless though, what is anyone supposed to learn from that without any clues of the cause or other diagnostic capabilities? Now there's literally *nothing* left for the devs to look into, as you put it 🙄

**Reply by enigmaticzebra** [reply]
I'd been trying for several days to fix it, although weirdly enough it's now working correctly after reinstalling so the issues probably with the install process somewhere. You do raise some good point's tho, should of contacted here first for support.
